packages/solid-query/src/__tests__/createQuery.test.tsx

```diff
@@ -3983,7 +3983,7 @@ describe('createQuery', () => {
 
     const query = queryCache.find({ queryKey: key })
     // @ts-expect-error
-    expect(query!.cacheTimeout).toBe(undefined)
+    expect(query!.gcTimeout).toBe(undefined)
```

Comment:
same here, this doesn't actually do anything.

Reply:
solved ✅